nuxeo-core/nuxeo-core-schema/src/main/resources/OSGI-INF/SchemaService.xml

```diff
@@ -150,6 +150,31 @@
       class="org.nuxeo.ecm.core.schema.SchemaBindingDescriptor" />
   </extension-point>
 
+  <extension-point name="deprecation">
+
+    <documentation>
+      Extension Point to register deprecated properties.
+      XML extensions may contain any number of 'property' elements of the form:
+      <code>
+        <property schema="common" name="size" deprecated="true" />
+        <property schema="file" name="filename" fallback="content/name" />
+      </code>
+      This means property size of schema common is now marked as deprecated in deprecation service.
+      This generates WARN message of usage to help to remove deprecated usage.
+      <p />
+      The fallback attribute is optional.
+      When presents, it is used to set the value to the fallback property and get value from fallback property when it
+      exists. Fallback xpath has to be in the same schema than deprecated property.
+      The deprecated attribute is optional, its default value is false (ie: removed property).
+      <p />
+      Removed properties are used to relax Nuxeo Platform behavior when there's remaining usage of property removed from
+      schema definition (for example 'file:filename').
```

Comment:
Reading this documentation, I don't understand the difference in behavior between a removed property and a deprecated property.